scene_graph/vrd_tutorial.py

```diff
@@ -26,38 +41,45 @@
 # - `source_img`: filename for the corresponding image the relationship is in
 # - `subject_bbox`: coordinates of the bounding box for the object `[ymin, ymax, xmin, xmax]`
 # - `subject_category`: category of the subject
-#
-# Note that the training DataFrame will have a labels field with all -1s. This denotes the lack of labels for that particular dataset. In this tutorial, we will assign probabilistic labels to the training set by writing labeling functions over attributes of the subject and objects!
 
-# %%
+# +
 # %load_ext autoreload
 # %autoreload 2
 
 import numpy as np
 
-# %%
+# -
+
+# If you are running this notebook for the first time, it will take ~15 mins to download all the required data. **Setting `sample = False` will download the full VRD dataset, which can take ~3 hours!**
+#
+# The sampled version of the dataset **uses the same 26 examples across the train, dev, and test sets. This setting is meant to demonstrate how Snorkel works with this task, NOT TO DEMONSTRATE PERFORMANCE.**
+#
+
+# +
 from scene_graph.utils import load_vrd_data
 
-train_df, valid_df, test_df = load_vrd_data()
+sample = True
+is_travis = "TRAVIS" in os.environ
+train_df, valid_df, test_df = load_vrd_data(sample, is_travis)
 
 print("Train Relationships: ", len(train_df))
 print("Dev Relationships: ", len(valid_df))
 print("Test Relationships: ", len(test_df))
+# -
+
+# Note that the training DataFrame will have a labels field with all -1s. This denotes the lack of labels for that particular dataset. In this tutorial, we will assign probabilistic labels to the training set by writing labeling functions over attributes of the subject and objects!
 
-# %% [markdown]
 # ## 2. Writing Labeling Functions
 # We now write labeling functions to detect what relationship exists between pairs of bounding boxes. To do so, we can encode various intuitions into the labeling functions. _Categorical_ intution: knowledge about the categories of subjects and objects usually involved in these relationships (e.g., `person` is usually the subject for predicates like `ride` and `carry`), and _spatial_ intuition: knowledge about the relative positions of the subject and objects (e.g., subject is usually higher than the object for the predicate `ride`).
```

Comment:
nit: can we break up _categorical intuition_ and _spatial intuition_ into bullets?